src/Uno.UI.Tests/BinderTests/Given_Binder.cs

```diff
@@ -884,7 +884,7 @@ public BaseTarget(BaseTarget parent = null)
 				DataContextChangedCount++;
 			}
 
-			#region ChildrenBinders DependencyProperty
+#region ChildrenBinders DependencyProperty
```

Comment:
This seems like unrelated changes ? Did you mean to add a unit test here ?

Reply:
Yes I did however the test i wanted to add tests a method not available in net461. I left the autoformatted code changes